src/components/lib/posix/posix.c

```diff
@@ -46,9 +49,9 @@ cos_write(int fd, const void *buf, size_t count)
 {
 	/* You shouldn't write to stdin anyway, so don't bother special casing it */
 	if (fd == 1 || fd == 2) {
-		sl_lock_take(&stdout_lock);
+		//sl_lock_take(&stdout_lock);
```

Comment:
I'm not sure that I want to remove all of these. Certainly we want to get rid of sl, but I don't think that we want to get rid of the code for lock take/release. We want to add it back in, so it is odd ot remove it all. How about this:

you define some #defines for lock_take lock_release and other similar functions. The #defines can be empty for now (thus not generating any executable logic).